plugin/vcs/gitlab/gitlab.go

```diff
@@ -311,7 +318,11 @@ func (provider *Provider) FetchRepositoryActiveMemberList(ctx context.Context, o
 		}
 	}
 
-	return activeRepositoryMemberList, nil
+	if len(emptyEmailUserIDList) != 0 {
+		return nil, fmt.Errorf("[ %v ] did not configured their public email, please make sure every members' public emails are configured before syncing, see https://docs.gitlab.com/ee/user/profile", strings.Join(emptyEmailUserIDList, ", "))
```

Comment:
```suggestion
		return nil, fmt.Errorf("[ %v ] did not configure their public email in GitLab, please make sure every members' public email is configured before syncing, see https://docs.gitlab.com/ee/user/profile", strings.Join(emptyEmailUserIDList, ", "))
```

Reply:
A better way is to create a wiki page https://github.com/bytebase/bytebase/wiki and use that link.